benchmark/sirun/get-results.js

```diff
@@ -38,7 +38,14 @@ function get (url, headers) {
 }
 
 async function getBuildNumsFromGithub (ref) {
-  const results = JSON.parse(await get(statusUrl(ref)))
+  const results = []
+  let page = 1
+  let reply = JSON.parse(await get(statusUrl(ref, page)))
+  results.push(...reply)
+  while (reply.length === 100) {
```

Comment:
`do...while`

Reply:
Also, maybe a bit late to ask this but why is GitHub even needed?

Reply:
> `do...while`

Sure.

> Also, maybe a bit late to ask this but why is GitHub even needed?

I had a pretty rough time trying to figure out how to get a list of jobs for a commit from the CircleCI API. Open to suggestions tho.

Reply:
Ah, yeah their API is not super well documented but this might help: https://github.com/DataDog/dd-trace-js/blob/1fc4a16c6c04fffde9933312737686da278d788e/scripts/prepublish.js#L49-L52.

Don't ask me where I found how to do the above 😆 But also if the current approach works it might not be worth it to invest time in using only CircleCI API.